### PR TITLE
Suggest people to applicant/recruiter

### PR DESCRIPTION
**Pull request by yongsokheng**

### CODE REVIEW

File: app/chewy/applicants_index.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          }},
+          {match: {
+            schools: {
+              query: applicant.educations.pluck(:school).join(" "),
+              boost: 5
+            }
+          }},
+          {match: {job_types: applicant.job_types.pluck(:name).join(" ")}},
+          {match: {skills: applicant.skills.pluck(:name).join(" ")}},
+          {match: {position: applicant.user_profile ? applicant.user_profile.current_position : ""}}
```
Comment (NgengChhengkim) [inline app/chewy/applicants_index.rb:36]: yong tang lakhan current position teat?

File: app/controllers/static_pages_controller.rb [hunk 2]
```diff
@@ -4,5 +4,18 @@ def home
     @job_types = JobType.all
     @post = Post.new
     @post.build_job_post
+
+    if current_user.recruiter?
+      recruiter_ids = RecruitersIndex.suggest_to_recruiter(current_user).map &:id
```
Comment (NgengChhengkim) [inline app/controllers/static_pages_controller.rb]: map &:id o ngoai

File: app/models/user_job_type.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class UserJobType < ApplicationRecord
+  belongs_to :user
+  belongs_to :job_type
```
Comment (NgengChhengkim) [inline app/models/user_job_type.rb:3]: uniq

File: app/chewy/applicants_index.rb (hunk 1 above)
Comment (yongsokheng) [reply app/chewy/applicants_index.rb:36]: dak tov ber mean srodeang khnea vea chenh mok

**Pull request merged**